situp-plugins/lmdb-processor-state/src/main/java/com/amazon/situp/plugins/processor/state/LmdbProcessorState.java

```diff
@@ -115,6 +117,14 @@ public void clear() {
         }
     }
 
+    public void delete() {
+        close();
```

Comment:
you can simplify this by deleting the entire directory. The parent directory contains both of these files.

Reply:
oh wait, I just saw the EnvFlags.MDB_NOSUBDIR.

Does this mean that there will be a lock file with a unique name per DB?

Reply:
talked offline